arquin/compiler/main.py

```diff
@@ -3,8 +3,8 @@
 from qiskit.compiler import transpile
 from qiskit.converters import circuit_to_dag, dag_to_circuit
 
-from compiler.converters import edges_to_source_graph, circuit_to_graph, write_source_graph_file, write_target_graph_file, edges_to_coupling_map
-from compiler.comms import distribute_gates, construct_local_circuits, assign_qubits, read_distribution_file
+from arquin.compiler.converters import edges_to_source_graph, circuit_to_graph, write_source_graph_file, write_target_graph_file, edges_to_coupling_map
+from arquin.compiler.comms import distribute_gates, construct_local_circuits, assign_qubits, read_distribution_file
```

Comment:
It's usually easier to read if we just `import arquin` at the top, then use the full function name wherever needed. So instead of calling `edges_to_source_graph` in the code, it would be `tarquin.converters.edges_to_source_graph`. It's also easier this way to know where the function is defined without having to track it down at the top amongst all the imports